client/src/components/modal/modalReview.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import moment from 'moment';
+import FriendNumber from './icons/friendNumber.jsx';
+import reviewNumber from './icons/reviewNumber.jsx';
+import styles from './modalReview.module.css';
+import ReviewNumber from './icons/reviewNumber.jsx';
+
+class ModalReview extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            displayFull: false
+        }
+        this.toggleBody = this.toggleBody.bind(this);
+    }
+
+    toggleBody() {
+        this.setState(state => ({ displayFull: !state.displayFull }))
+    }
+    render() {
+        let { name, friendNumber, reviewNumber, body, snippet, dish, stars, date, userAvatar } = this.props;
+        let formattedName;
+        if (name) {
+            let names = name.split(' ');
+            formattedName = `${names[0]} ${names[1].charAt(0)}.`;
+        } else {
+            formattedName = 'Anonymous';
+        }
+        let starsArray = [0, 1, 2, 3, 4];
+
+        return (
+            <div className={styles.container}>
+                <img src={userAvatar} className={styles.avatar} />
+                <div className={styles.name}>{formattedName}</div>
+                <div className={styles.friends}>
+                    <FriendNumber /> {friendNumber}
+                </div>
+                <div className={styles.reviews}>
+                    <ReviewNumber fill='rgb(241, 92, 0)' /> {reviewNumber}
+                </div>
+                <div className={styles.stars}>stars: {starsArray.map((data, index) => {
+                    return (
+                        <ReviewNumber fill={stars === 5 ? 'rgb(211,35,34)'
```

Comment:
Commenting just to say this looks really cool and I should consider using this when I need to do similar things next time.